numpy/linalg/linalg.py

```diff
@@ -2006,7 +2006,7 @@ def _multi_svd_norm(x, row_axis, col_axis, op):
     """
     if row_axis > col_axis:
         row_axis -= 1
-    y = rollaxis(rollaxis(x, col_axis, x.ndim), row_axis, -1)
+    y = moveaxis(x, [row_axis, col_axis], [-2, -1])
```

Comment:
Ah yes, I get it now: If `if row_axis > col_axis`, and `col_axis` is moved to the end of the shape first, one needs to `row_axis -= 1` to still get the right axis when moving the second one to the end.

Indeed, we don't need this anymore. Praised be `moveaxis`!